lib/Doctrine/Persistence/Mapping/Driver/FileDriver.php

```diff
@@ -126,10 +127,10 @@ public function getAllClassNames()
             return (array) $this->locator->getAllClassNames($this->globalBasename);
         }
 
-        return array_merge(
+        return array_unique(array_merge(
```

Comment:
Does the problem only occur with thr `FileDriver`?

Reply:
Yes, I just checked and this is the only place in the project where `array_merge` is used with indexed/numeric arrays without `array_unique`.